link-admin/.storybook/config.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
-import { Provider as FelaProvider } from 'react-fela'
+import { RendererProvider as FelaProvider } from 'react-fela'
 import { configure, addDecorator } from '@storybook/react'
 
 import { createRenderer } from 'fela'
 
 import '../src/index.css'
-import '../src/utils/globalStyles'
+// import '../src/utils/globalStyles'
```

Comment:
Let's just remove this line and the entire file instead of hanging some comments out to dry

Reply:
CSS imports have been removed